app/Helpers/database/player-achievement.php

```diff
@@ -187,7 +188,7 @@ function getRecentUnlocksPlayersData(
     }
 
     // Get recent winners, and their most recent activity:
-    $query = "SELECT u.User, u.RAPoints, " . unixTimestampStatement('pa.unlocked_at', 'DateAwarded') . "
+    $query = "SELECT u.User, u.RAPoints, u.RASoftcorePoints, " . unixTimestampStatement('pa.unlocked_at', 'DateAwarded') . "
```

Comment:
As best as I can tell, this function is only called by the Connect API. What is the reasoning for this change?

Reply:
I can roll that back if desired. I was just trying to make sure it was in parity

Reply:
I see, thank you for that clarification 👍

To the best of my knowledge, no emulator consumer is clamoring for this value, and we want to encourage non-emulator consumers who are using the Connect API to switch over to the Web API.

I lean towards this being rolled back and the branch's changeset being scoped specifically to the Web API.

Reply:
Cool. Will change it back

Reply:
Done